pkg/importer/spanner/import.arrai

```diff
@@ -0,0 +1,18 @@
+let import = \importSql \appName \syslPackage
+    let spanner = //{./spanner};
+    let sysl = //{./sysl};
+    let stmts = spanner.parseSchema([importSql]);
+    let model = spanner.applyStmt(stmts, spanner.emptyModel);
+    $'${sysl.transformModel(
+        cond {
+            (model.schema): model,
+            _: (
+                entities: model.entities,
+                indexes: model.indexes,
+                schema: {(name: appName)},
+            )
+        }, syslPackage)}';
```

Comment:
`sysl.transformModel` is already a string.
```suggestion
    sysl.transformModel(
        cond {
            (model.schema): model,
            _: (
                entities: model.entities,
                indexes: model.indexes,
                schema: {(name: appName)},
            )
        }, syslPackage);
```

Reply:
fine